From f42f70fa9a3a82031d6d89c33c975d13f87a8684 Mon Sep 17 00:00:00 2001
From: liutianyu1250 <liutianyu1250@phytium.com.cn>
Date: Mon, 5 Feb 2024 09:52:51 +0800
Subject: [PATCH 06/67] BACKPORT: PHYTIUM: Update phytium copyright info to
 2024

Signed-off-by: liutianyu1250 <liutianyu1250@phytium.com.cn>

Link: https://gitee.com/phytium_embedded/phytium-linux-kernel/commit/f1b4bd2a3b847e061631dfea27f4c64bb75cfe9b
[Kexy: Resolved minor conflicts in
 arch/arm64/boot/dts/phytium/pe2201-demo-ddr4.dts,
 arch/arm64/boot/dts/phytium/pe2201.dtsi,
 arch/arm64/boot/dts/phytium/pe2202-chillipi-edu-board.dts,
 arch/arm64/boot/dts/phytium/pe2202-demo-ddr4-local.dts,
 arch/arm64/boot/dts/phytium/pe2202-demo-ddr4.dts,
 arch/arm64/boot/dts/phytium/pe2202-miniitx-board.dts,
 arch/arm64/boot/dts/phytium/pe2202-power-board.dts,
 arch/arm64/boot/dts/phytium/pe2202.dtsi,
 arch/arm64/boot/dts/phytium/pe2204-come-board.dts,
 arch/arm64/boot/dts/phytium/pe2204-demo-ddr4-local.dts,
 arch/arm64/boot/dts/phytium/pe2204-demo-ddr4.dts,
 arch/arm64/boot/dts/phytium/pe2204-edu-board.dts,
 arch/arm64/boot/dts/phytium/pe2204-hanwei-board.dts,
 arch/arm64/boot/dts/phytium/pe2204-miniitx-board.dts,
 arch/arm64/boot/dts/phytium/pe2204-vpx-board.dts,
 arch/arm64/boot/dts/phytium/pe2204.dtsi,
 arch/arm64/boot/dts/phytium/pe220x.dtsi,
 arch/arm64/boot/dts/phytium/phytiumpi_firefly.dts,
 drivers/char/hw_random/phytium-rng.c,
 drivers/char/ipmi/bt_bmc_phytium.c,
 drivers/char/ipmi/kcs_bmc_phytium.c,
 drivers/dma/phytium/phytium-ddmac.c,
 drivers/dma/phytium/phytium-ddmac.h,
 drivers/edac/phytium_pe220x_edac.c, drivers/gpio/gpio-phytium-core.c,
 drivers/gpio/gpio-phytium-core.h, drivers/gpio/gpio-phytium-pci.c,
 drivers/gpio/gpio-phytium-platform.c,
 drivers/gpio/gpio-phytium-sgpio.c, drivers/gpu/drm/phytium/pe220x_dc.c,
 drivers/gpu/drm/phytium/pe220x_dc.h,
 drivers/gpu/drm/phytium/pe220x_dp.c,
 drivers/gpu/drm/phytium/pe220x_dp.h,
 drivers/gpu/drm/phytium/pe220x_reg.h,
 drivers/gpu/drm/phytium/phytium_crtc.c,
 drivers/gpu/drm/phytium/phytium_crtc.h,
 drivers/gpu/drm/phytium/phytium_debugfs.c,
 drivers/gpu/drm/phytium/phytium_debugfs.h,
 drivers/gpu/drm/phytium/phytium_display_drv.c,
 drivers/gpu/drm/phytium/phytium_display_drv.h,
 drivers/gpu/drm/phytium/phytium_dp.c,
 drivers/gpu/drm/phytium/phytium_dp.h,
 drivers/gpu/drm/phytium/phytium_fb.c,
 drivers/gpu/drm/phytium/phytium_fb.h,
 drivers/gpu/drm/phytium/phytium_fbdev.c,
 drivers/gpu/drm/phytium/phytium_fbdev.h,
 drivers/gpu/drm/phytium/phytium_gem.c,
 drivers/gpu/drm/phytium/phytium_gem.h,
 drivers/gpu/drm/phytium/phytium_panel.c,
 drivers/gpu/drm/phytium/phytium_panel.h,
 drivers/gpu/drm/phytium/phytium_pci.c,
 drivers/gpu/drm/phytium/phytium_pci.h,
 drivers/gpu/drm/phytium/phytium_plane.c,
 drivers/gpu/drm/phytium/phytium_plane.h,
 drivers/gpu/drm/phytium/phytium_platform.c,
 drivers/gpu/drm/phytium/phytium_platform.h,
 drivers/gpu/drm/phytium/phytium_reg.h,
 drivers/gpu/drm/phytium/px210_dc.c, drivers/gpu/drm/phytium/px210_dc.h,
 drivers/gpu/drm/phytium/px210_dp.c, drivers/gpu/drm/phytium/px210_dp.h,
 drivers/gpu/drm/phytium/px210_reg.h, drivers/hwmon/tacho-phytium.c,
 drivers/hwspinlock/phytium_hwspinlock.c,
 drivers/i2c/busses/i2c-phytium-common.c,
 drivers/i2c/busses/i2c-phytium-core.h,
 drivers/i2c/busses/i2c-phytium-master.c,
 drivers/i2c/busses/i2c-phytium-pci.c,
 drivers/i2c/busses/i2c-phytium-platform.c,
 drivers/i2c/busses/i2c-phytium-slave.c, drivers/iio/adc/phytium-adc.c,
 drivers/input/keyboard/phytium-keypad.c,
 drivers/input/serio/phytium-ps2.c,
 drivers/irqchip/irq-gic-phytium-2500-its.c,
 drivers/irqchip/irq-gic-phytium-2500.c,
 drivers/irqchip/irq-phytium-ixic.c, drivers/mailbox/phytium_mailbox.c,
 drivers/media/platform/phytium/phytium_jpeg_core.c,
 drivers/media/platform/phytium/phytium_jpeg_core.h,
 drivers/media/platform/phytium/phytium_jpeg_reg.h,
 drivers/mfd/phytium_px210_i2s_lsd.c,
 drivers/mfd/phytium_px210_i2s_mmd.c,
 drivers/mmc/host/phytium-mci-pci.c,
 drivers/mmc/host/phytium-mci-plat.c, drivers/mmc/host/phytium-mci.c,
 drivers/mmc/host/phytium-mci.h, drivers/mmc/host/phytium-sdci.c,
 drivers/mmc/host/phytium-sdci.h, drivers/mtd/nand/raw/phytium_nand.c,
 drivers/mtd/nand/raw/phytium_nand.h,
 drivers/mtd/nand/raw/phytium_nand_pci.c,
 drivers/mtd/nand/raw/phytium_nand_plat.c,
 drivers/net/can/phytium/phytium_can.c,
 drivers/net/can/phytium/phytium_can.h,
 drivers/net/can/phytium/phytium_can_pci.c,
 drivers/net/can/phytium/phytium_can_platform.c,
 drivers/pci/controller/pcie-phytium-ep.c,
 drivers/pci/controller/pcie-phytium-ep.h,
 drivers/pci/controller/pcie-phytium-register.h,
 drivers/pwm/pwm-phytium.c, drivers/remoteproc/homo_remoteproc.c,
 drivers/spi/spi-phytium-dma.c, drivers/spi/spi-phytium-pci.c,
 drivers/spi/spi-phytium-plat.c, drivers/spi/spi-phytium-qspi.c,
 drivers/spi/spi-phytium.c, drivers/spi/spi-phytium.h,
 drivers/tty/serial/phytium-uart.c, drivers/w1/masters/phytium_w1.c,
 include/linux/irqchip/arm-gic-phytium-2500.h,
 sound/pci/hda/hda_phytium.c, sound/pci/hda/hda_phytium.h,
 sound/soc/codecs/es8336.c, sound/soc/codecs/es8336.h,
 sound/soc/codecs/es8388.c, sound/soc/phytium/local.h,
 sound/soc/phytium/phytium_i2s.c, sound/soc/phytium/pmdk_dp.c,
 sound/soc/phytium/pmdk_es8336.c, and sound/soc/phytium/pmdk_es8388.c]
Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c b/drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c
index 86f491ea569e..a930cea6280b 100644
--- a/drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c
+++ b/drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c
@@ -2,7 +2,7 @@
 /*
  * Phytium DWMAC specific glue layer
  *
- * Copyright (C) 2022, Phytium Technology Co., Ltd.
+ * Copyright (c) 2022-2024 Phytium Technology Co., Ltd.
  *
  * Chen Baozi <chenbaozi@phytium.com.cn>
  */
--
2.50.1
